changelog/april2024/2024-04-04-cockpit-added-metrics-degrissivity.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Metrics degrissivity
+status: added
+author:
+    fullname: 'Join the #[xxx] channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-04-04
+category: observability
+product: cockpit
+---
+
+We are pleased to announce degressive pricing on Cockpit external metrics.
+
+We add 6 levels for degressive monthly pricing : 
+- 0 to 10B metrics : 0.15€/ Million samples
+- 10 to 25B metrics : 0.13€/ Million samples
+- 25B to 50B metrics : 0.12€/ Million samples
+- 50 to 100B metrics : 0.11€/ Million samples
+- 100 to 200B metrics : 0.10€/ Million samples
+- 200B and more : 0.08€/ Million samples
+
+Pricing is applied **in its level**. So if you send 52 Billion external samples a month, you'll be billed as so :
+- 10000*0,15 : 1500€
+- 15000*0,13 : 1950€
+- 25000*0,12 : 3000€
+- 2000*0,11 : 220€
+
+Total : 6670€ instead of 7800€
```

Comment:
```suggestion
Check out our latest update: volume discounts for your external metrics, allowing you to enjoy a cost-efficient monitoring solution!

**How it works:**
The monthly pricing for external metrics will be calculated in levels. **Scaleway applies six levels of volume discounts.**

| Levels       | Range                                    | Monthly pricing with volume discounts    |
|--------------|------------------------------------------|------------------------------------------|
| Level one    | From 0 to 10 billion external metrics    | €0.15 per million samples                |
| Level two    | From 10 to 25 billion external metrics   | €0.13 per million samples                |
| Level three  | From 25 to 50 billion external metrics   | €0.12 per million samples                |
| Level four   | From 50 to 100 billion external metrics  | €0.11 per million samples                |
| Level five   | From 100 to 200 billion external metrics | €0.10 per million samples                |
| Level six    | From 200 billion external metrics and beyond              | €0.08 per million samples                |


Here is an example of how you would be billed for **sending 52 billion external metrics samples per month**:

- 10,000 million samples of metrics x €0.15: €1.500
- 15,000 million samples of metrics x €0.13: €1.950
- 25,000 million samples of metrics x €0.12: €3.000
- 2 million samples of metrics x €0.11: €2.200

**Total:** €6.650 per month with volume discounts instead of €7.800 without volume discounts.

Upgrade your monitoring experience with Cockpit and enjoy smarter pricing that scales with your needs!
```